core/templates/pages/volunteer-page/volunteer-page.component.spec.ts

```diff
@@ -71,8 +71,10 @@ describe('Volunteer page', () => {
     spyOn(translateService.onLangChange, 'subscribe');
     component.ngOnInit();
 
-    expect(component.mapImgPath).toBe(
-      '/volunteer/map.png');
+    expect(component.bannerImgPath).toBe(
+      '/volunteer/banner-background.png');
+    expect(component.footerImgPath).toBe(
+      '/volunteer/footer-background.png');
```

Comment:
Can you add a pre-check for this?

Reply:
Sorry If I wasn't clear. I meant to check if `bannerImgPath` and `footerImgPath` is defined before `component.ngOnInit();` is called